packages/backend/src/quickpicks.ts

```diff
@@ -0,0 +1,74 @@
+import type { History } from './history';
+import * as extensionApi from '@podman-desktop/api';
+import * as os from 'node:os';
+import { resolve } from 'node:path';
+
+export interface BootcBuildOptionSelection {
+  type: string;
+  folder: string;
+  path: string;
+  arch: string;
+}
+
+export async function bootcBuildOptionSelection(history: History): Promise<BootcBuildOptionSelection | undefined> {
+  const selection = await extensionApi.window.showQuickPick(
+    [
+      { label: 'QCOW2', detail: 'QEMU image (.qcow2)', format: 'qcow2' },
+      { label: 'AMI', detail: 'Amazon Machine Image (.ami)', format: 'ami' },
```

Comment:
I think it might work better having a Class with history being a class field

and then you call the method to get the build options

Reply:
I thought of that too, but then I realized that we'd have to pass in the extensionContext in order to get the storagePath information, so it'd be trading one thing for another.